### PR TITLE
Implement method used to find all the Power of Attorney Data

### PR DESCRIPTION
**Pull request by aroltsch**
connects #26 

```
# Instantiate 
bgs = BGS::Services.new(env: ENV["BGS_ENVIRONMENT"], client_ip: "127.0.0.1", client_station_id: "283",client_username: "caseflow1", application: "caseflow", ssl_cert_key_file: ENV["BGS_KEY_LOCATION"],ssl_cert_file: ENV["BGS_CERT_LOCATION"], ssl_ca_cert: ENV["BGS_CA_CERT_LOCATION"], log: true)
# Run 
bgs.data.find_power_of_attorneys`
```
Sample Response:
```
[{:legacy_poa_cd=>"002", :nm=>"MAINE DEPARTMENT OF VETERANS SERVICES", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"46004"},
 {:legacy_poa_cd=>"003", :nm=>"POLISH LEGION OF AMERICAN VETERANS USA", :org_type_nm=>"POA National Organization", :ptcpnt_id=>"46028"},
 {:legacy_poa_cd=>"004", :nm=>"RHODE ISLAND DIVISION OF VETERANS AFFAIRS", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"46044"},
 {:legacy_poa_cd=>"005", :nm=>"VERMONT VETERANS AFFAIRS SECTION, MILITARY DEPT", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"46053"},
 {:legacy_poa_cd=>"006", :nm=>"NEW YORK DIVISION OF VETERANS AFFAIRS", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"46018"},
 {:legacy_poa_cd=>"007", :nm=>"THE RETIRED ENLISTED ASSOCIATION", :org_type_nm=>"POA National Organization", :ptcpnt_id=>"46043"},
 {:legacy_poa_cd=>"008", :nm=>"CONNECTICUT DEPARTMENT OF VETERANS AFFAIRS", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"45981"},
 {:legacy_poa_cd=>"009", :nm=>"NEW JERSEY DEPT OF MILITARY AND VETERANS' AFFAIRS", :org_type_nm=>"POA State Organization", :ptcpnt_id=>"46016"},
 {:legacy_poa_cd=>"00A", :nm=>"MARK R. CALDWELL", :org_type_nm=>"POA Attorney", :ptcpnt_id=>"46038"},
 {:legacy_poa_cd=>"00B", :nm=>"KENNETH M. CARPENTER", :org_type_nm=>"POA Attorney", :ptcpnt_id=>"46037"},
 {:legacy_poa_cd=>"00C", :nm=>"STEPHEN L. DEVITA", :org_type_nm=>"POA Attorney", :ptcpnt_id=>"46039"},
 {:legacy_poa_cd=>"00D", :nm=>"JENNIFER H WILL", :org_type_nm=>"POA Attorney", :ptcpnt_id=>"600116504"},
 {:legacy_poa_cd=>"00E", :nm=>"LEE R FINSTAD", :org_type_nm=>"POA Attorney", :ptcpnt_id=>"45998”}]
```

### CODE REVIEW

**Comment by amprokop** [comment]
@aroltsch — awesome! Is that all the POA data that this endpoint gives us in UAT or did you shorten the response? My impression was that the result should be much much longer.

@shanear @nicholasholtz — any idea why the POA code would be named `legacy_poa_cd`? I'm assuming it means POA legacy code. I wonder if they're trying to phase this out..

**Comment by aroltsch** [comment]
@amprokop I shortened the response. It was huuge

**Comment by amprokop** [comment]
@shanear i don't think i can approve this, can you?

File: lib/bgs/base.rb [hunk 1]
```diff
@@ -109,7 +109,7 @@ def client
     end
 
     # Proxy to call a method on our web service.
-    def request(method, message)
+    def request(method, message = nil)
```
Comment (amprokop) [inline lib/bgs/base.rb]: this is because we send no parameter to the find poa endpoint right?

File: lib/bgs/services/standard_data.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+module BGS
+  # This service is used to find Standard Data from Share.
+  class StandardDataWebService < BGS::Base
+    def self.service_name
+      "standard_data"
+    end
+
+    # This method is used to find all the Power of Attorney Data.
+    def find_power_of_attorneys
```
Comment (amprokop) [inline lib/bgs/services/standard_data.rb]: this sounds so awkward but i guess we should try to match the endpoint names that exist

File: lib/bgs/base.rb (hunk 1 above)
Comment (aroltsch) [inline lib/bgs/base.rb]: yes.

File: lib/bgs/services/standard_data.rb (hunk 2 above)
Comment (aroltsch) [inline lib/bgs/services/standard_data.rb]: yep, that's what we have been doing.

File: lib/bgs/services/standard_data.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# public domain within the United States.
+#
+# Additionally, we waive copyright and related rights in the work
+# worldwide through the CC0 1.0 Universal public domain dedication.
+
+module BGS
+  # This service is used to find Standard Data from Share.
+  class StandardDataWebService < BGS::Base
+    def self.service_name
+      "standard_data"
```
Comment (shanear) [inline lib/bgs/services/standard_data.rb]: maybe just shorten this to `data`?

Comment (aroltsch) [inline lib/bgs/services/standard_data.rb]: i like that
